docs/src/pages/[platform]/theming/default-theme/icons/Colors.tsx

```diff
@@ -0,0 +1,26 @@
+import { Icon } from '@aws-amplify/ui-react';
+
+export function Colors() {
+  return (
+    <Icon ariaLabel="" viewBox={{ width: 128, height: 128 }}>
```

Comment:
Oh that's awesome. I was thinking you get the icons somewhere else but you built them by yourself